backend/models/material.model.ts

```diff
@@ -0,0 +1,34 @@
+import mongoose, { Document, Schema } from "mongoose";
+
+// const Currency = {
+// 	UkCurrency: "GBP",
+// 	UsCurrency: "USD",
+// } as const;
+
+// type CurrencyType = typeof Currency[keyof typeof Currency];
+
+// export type IMaterial = {
+// 	// supplierId: String;
+// 	name: String;
+// 	price: Number;
+// 	// currency: CurrencyType;
+// 	description?: String;
+// };
+
+// export interface MaterialModel extends IMaterial, Document {}
+
+const materialSchema = new Schema({
+	// supplierId: { type: String, required: true },
+	name: { type: String, required: true },
+	price: { type: Number, required: true },
+	description: { type: String },
+	// currency: {
```

Comment:
removing the currency here means you would need to make a decision of sticking to one currency only - so GBP